diff --git a/src/dots/DigitalOceanApiKeyAuthenticationProvider.ts b/src/dots/DigitalOceanApiKeyAuthenticationProvider.ts
index ebd662ad7442248526661a2db584bced62d00406..563d2cdbd689b12eb223c97c274f871919c39b6b 100644
--- a/src/dots/DigitalOceanApiKeyAuthenticationProvider.ts
+++ b/src/dots/DigitalOceanApiKeyAuthenticationProvider.ts
@@ -1,6 +1,5 @@
-import { AuthenticationProvider, RequestInformation } from "@microsoft/kiota-abstractions";
-// @ts-expect-error: Importing JSON for dynamic version in User-Agent header
-import pkg from '../../package.json' assert { type: "json" };
+import { type AuthenticationProvider, RequestInformation } from "@microsoft/kiota-abstractions";
+import pkg from '../../package.json' with { type: "json" };
 
 /** Authenticate a request by using an API Key */
 export class DigitalOceanApiKeyAuthenticationProvider implements AuthenticationProvider {
